features/frameworks/utest/utest/utest_types.h

```diff
@@ -334,8 +374,11 @@ namespace v1 {
 
 
     // deprecations
-    __deprecated_message("Use CaseRepeatAll instead.")     const control_t CaseRepeat            = CaseRepeatAll;
-    __deprecated_message("Use CaseRepeatHandler instead.") const control_t CaseRepeatHandlerOnly = CaseRepeatHandler;
+    __deprecated_message("Use CaseRepeatAll instead.")
+    extern const base_control_t CaseRepeat;
```

Comment:
Maybe a different pr, but should we update these to use MBED_DEPRECATED instead?